### PR TITLE
add body parser for POST requests

### PR DESCRIPTION
**Pull request by suryadutta**

### CODE REVIEW

**Comment by kljensen** [comment]
@suryadutta you think we should include this so they don't have to figure it out?

**Comment by suryadutta** [comment]
I imagined it would be somewhat confusing since it used to be part of the default express framework, and a lot of documentation references req.body. It's definitely something that people can figure out with enough time.

File: server.js [hunk 1]
```diff
@@ -22,8 +23,14 @@ nunjucks.configure('views', {
     autoescape: true,
     express: app,
 });
+
 app.set('view engine', 'html');
 
+app.use(bodyParser.json());
```
Comment (kljensen) [inline server.js]: @suryadutta Do they need the json body parsing?

Comment (suryadutta) [reply server.js]: just tested it - no it's not. I can setup a different pull request without this if it would be easier

**Comment by kljensen** [comment]
I think you can update this one. Then I'll merge. Thanks, Kyle

**Comment by kljensen** [comment]
Thanks for closing 😅

**Pull request closed without merging**